pkg/kubelet/server/stats/volume_stat_calculator.go

```diff
@@ -131,21 +167,92 @@ func (s *volumeStatCalculator) calcAndStoreStats() {
 }
 
 // parsePodVolumeStats converts (internal) volume.Metrics to (external) stats.VolumeStats structures
-func (s *volumeStatCalculator) parsePodVolumeStats(podName string, pvcRef *stats.PVCReference, metric *volume.Metrics) stats.VolumeStats {
+func (s *volumeStatCalculator) parsePodVolumeStats(podName string, pvcRef *stats.PVCReference, metric *volume.Metrics, volSpec v1.Volume) stats.VolumeStats {
```

Comment:
If all you need is the `FSType`, why not just pass in the `FSType`, instead of the whole volume spec?

Reply:
fixed